docs/tutorials/p2p.md

```diff
@@ -49,52 +48,43 @@ We will get into each of these below.
 ## The \_process() Function
 ==}
 
-We'll also need to set `read_p2p_packet()` in our `_process()` function so it is always looking for new packets:
+We'll also need to set `read_message()` in our `_process()` function so it is always looking for new packets:
 
 ```gdscript
 func _process(_delta) -> void:
 	Steam.run_callbacks()
 
 	# If the player is connected, read packets
 	if lobby_id > 0:
-		read_p2p_packet()
+		read_messages()
 ```
 
 If you are using the `global.gd` autoload singleton then you can omit the `run_callbacks()` command as they'll be running already.
 
-Here is a nice bit of code from **tehsquidge** for handling packet reading:
+Here is a nice bit of code from **Toi Lanh** based on **tehsquidge**'s networking code for handling packet reading:
```

Comment:
There is no packet limiting here though.  This should be removed entirely as it is shown above.